### PR TITLE
Rg diettracker

### PR DESCRIPTION
**Pull request by rgoolishian**
This branch was created off of Kurt's last changes so if I understand correctly, merging this will merge all of Kurt's changes along with mine to the master branch.

### CODE REVIEW

**Comment by ghost** [comment]
The Parser looks great to me. We have a lot of data to play with. Could really make this complex if we wanted.;)

File: .idea/libraries/hamcrest_core_1_3.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="hamcrest-core-1.3">
```
Comment (smarks) [inline .idea/libraries/hamcrest_core_1_3.xml:2]: DO NOT check ./idea director in git

File: src/main/java/edu/uml/diet/BasicFood.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package edu.uml.diet;
+
+/**
+ * Created by Kurt Johnson on 2/8/2015.
+ */
+public class BasicFood {
+    private String name;
+    private int calories;
+
+
```
Comment (smarks) [inline src/main/java/edu/uml/diet/BasicFood.java:10]: Needs JavaDoc

File: src/main/java/edu/uml/diet/DbParser.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+/**
+ * Created by Raymond on 2/15/2015.
+ */
+public class DbParser {
+
+    private InputStreamReader reader;
+
+    // container class for all food properties
+    public class dbFood{
+        public int NDB_No;
```
Comment (smarks) [inline src/main/java/edu/uml/diet/DbParser.java:15]: Yikes what this? 
No document in class so I can't tell. 
public member variables not a good idea. 
use whole word variable names.

**Comment by smarks** [comment]
You should not work of someone else's branch. You should work on what is in master

**Comment by rgoolishian** [comment]
Should we merge the original updates that Kurt made with master? Then I can recreate my branch off of master, clean up my code, and recreate the pull request?

**Comment by ghost** [comment]
Merging seams good. It's getting kinds crazy with all the branches. Didn't know how long we should wait in case Adil wanted to comment. -KJ

**Comment by sparky005** [comment]
I've looked over everything pretty quickly and it looks good. I think merging asap would be a good idea so we don't get too confused/messy. Ray, that seems like the best way to handle it

**Pull request closed without merging**